man/standard-options.xml

```diff
@@ -35,6 +35,16 @@
     <listitem><para>Do not query the user for authentication for privileged operations.</para></listitem>
   </varlistentry>
 
+  <varlistentry id='legend'>
+    <term><option>--legend</option></term>
+    <term><option>--no-legend</option></term>
```

Comment:
hmm, instead of adding both, maybe just replace --no-legend with --legend=yes/no. I think this is much nicer I think sinceit means: one parameter for one internal variable, means fewer options to list in the man page (as we can then hide --no-legend) and future proof (in case --legend=auto or --legency=on-tty or so one day shall be added). 

Given that this isn't the most often used switch I think doing this via the relatively verbose =yes/no syntax is OK even if longer. It also make stuff like this from shell scripts easier: `systemctl --legend=$LEGEND …` with any typical $LEGEND value...